### PR TITLE
Add intergattion test for provider service

### PR DESCRIPTION
**Pull request by josephbate**
# Pull Requests Requirements

- [ ] The PR title includes a brief description of the work done, including the
      Issue number if applicable.
- [ ] The PR includes a video showing the changes for the work done.
- [ ] The PR title follows conventional commit label standards.
- [ ] The changes confirm to the OpenElis Global x3 Styleguide and design
      documentation.
- [ ] The changes include tests or are validated by existing tests.
- [ ] I have read and agree to the Contributing Guidelines of this project.

## Summary

- been trying to create a provider service test but i have PSQLException blocker, apparently the fields of active and desynchronized which hold a boolean value have been limited with one character as an entry  which make true or false to be to long and therefore causing  a BatchUpdateException hence rejecting data
error:https://pastebin.com/YNFcL2WT

- also tried to look into the `hibernate/hbm/Provider.hbm.xml`  but it checks out fine  still wondering where the problem might be for these fields to be 1 character 

- i also notice still in the  `hibernate/hbm/Provider.hbm.xml`  the field of `PROVIDER_TYPE` is also limited to with a lenght of  1 i fear that too can cause errors

Cc. @mozzy11 
## Screenshots

[Add relevant screenshots here if applicable]

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## Other

[Add any additional information or notes here]

### CODE REVIEW

File: src/test/java/org/openelisglobal/provider/ProviderServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        providerService.deleteAll(providerService.getAll());
+    }
+
+    // @Test
+    // public void createProvider_shouldReturnCreatedProvider() throws Exception{
+    // String firstName = "John";
```
Comment (mherman22) [inline src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: did you intend to work on this but forgot?

**Review by mherman22 — CHANGES_REQUESTED** [review]

File: src/test/java/org/openelisglobal/provider/ProviderServiceTest.java (hunk 1 above)
Comment (mherman22) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: which is why the build is failing as can be seen at https://github.com/I-TECH-UW/OpenELIS-Global-2/actions/runs/11707145829/job/32605834580?pr=1302#step:8:10974

Comment (josephbate) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: Hello @mherman22 
i think can be resolved by updating this specific branch but that doesn't solve the problem i found out earlier on

Comment (mherman22) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: Can you click this link and see what its saying https://github.com/I-TECH-UW/OpenELIS-Global-2/actions/runs/11834718253/job/32975979073?pr=1302#step:8:11003

Comment (josephbate) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: yeah your right let me look into it shortly

Comment (mherman22) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: Is the PR title right? adding an integration test for provider server? or provider test?

Comment (josephbate) [reply src/test/java/org/openelisglobal/provider/ProviderServiceTest.java:33]: thank you for that correction @mherman22

**Pull request closed without merging**